### ISSUE TITLE
If I only have plex, the app errors out that I didn't set my Jellyfin URL

### ISSUE THREAD

**Issue by joshkerr** [mov-cli/mov-cli-jellyplex#6]
It should skip Jelly and look for plex.

**Comment by Nairooolf** [comment]
same here

**Comment by THEGOLDENPRO** [comment]
@r3tr0ananas could you possibly have a look at this, I think this is completely stopping the use of the plex scraper

**Comment by ananasmoe** [comment]
can't reproduce this issue.

**Comment by THEGOLDENPRO** [comment]
With your jellyfin environment variables unset too?

I just have a suspicion with that this line is causing it because as far as I remember scraper's init functions are executed no matter if you're using that scraper or not and `self.__auth` just happen to raise an exception if envs are missing.
https://github.com/mov-cli/mov-cli-jellyplex/blob/819780038c4263763c23dbaf421742146a2e0797/mov_cli_jellyplex/jellyfin/scraper.py#L40

sorry that I'm asking you to look into it, I'm just really busy

**Comment by ananasmoe** [comment]
yep, my jellyfin env is unset and iirc init functions only are executed when you use that scraper.

**Comment by ananasmoe** [comment]
@joshkerr what command do you use?

**Comment by rogers1lt** [comment]
I have just started using this plugin this afternoon with the similar error when using the command mov-cli -s jelly "query": 
Error: You haven't set the 'JELLY_URL' env!

**Comment by ananasmoe** [comment]
You need to do: `mov-cli -s jelly.plex "query"`

**Comment by rogers1lt** [comment]
That resolved my issue. I would request updating the documentation: [Usage](https://github.com/mov-cli/mov-cli-jellyplex?tab=readme-ov-file#usage-%EF%B8%8F)

**Comment by THEGOLDENPRO** [comment]
or you can also override the jelly namespace using the config like this:
```toml
[mov-cli.scrapers]
jelly = "jellyplex.plex"

[mov-cli.plugins]
jellyplex = "mov-cli-jellyplex"
```
more on that in the wiki: https://github.com/mov-cli/mov-cli/wiki/Configuration#namespace-overriding

now `-s jelly` should point to the plex scraper and you can use this command like normal, but it's up to you:
```sh
mov-cli -s jelly "query"
```

**Comment by joshkerr** [comment]
Did not resolve my issue. Its still erroring out for me. I checked that my plex environment variables are set and I do have my mov-cli config loading the plugin into the jelly namespace, but I'm still getting this output:

mov-cli -s jelly.plex ghost

Good Evening, joshkerr.
    It's 08:54 PM on a gorgeous Sunday! 

[INFO] (mov_cli) - Using 'jelly.plex' scraper...
[INFO] (mov_cli) - Searching for 'ghost'...
[INFO] (mov_cli) - [ATNS] ❌ Error occurred while searching with 'jelly.plex'! Trying the next scraper...

**Comment by rogers1lt** [comment]
That's the same new error message that I have. I didn't include that because it may be a Plex connection issue and not the original error message.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by joshkerr** [comment]
Actually, that did fix my issue. I was working in a terminal that didn't have my env variables set because it wasn't refreshed. Once I did that, it worked fine.

**Comment by joshkerr** [comment]
Try turning debug=true in your settings for mov-cli and it will show you why its erroring out. That's how I figured it out.
